packages/webcomponents/src/components/payments-list/payment.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Api, FetchPaymentsResponseType } from '../../api';
+import { config } from '../../../config';
+
+export interface IPaymentService {
+  fetchPayments(
+    accountId: string,
+    authToken: string,
+    params: any
+  ): Promise<FetchPaymentsResponseType>;
+}
+
+export class PaymentService implements IPaymentService {
+  async fetchPayments(
+    accountId: string,
+    authToken: string,
+    params: any
+  ): Promise<FetchPaymentsResponseType> {
```

Comment:
If you decide to implement the changes in my other comments - just need to make sure to update: `Promise:<FetchPaymentsResponseType>` in this file to `Promise<IApiResponseCollection<IPayment[]>>`